packages/prettier-config/package.json

```
@@ -18,7 +18,7 @@
     "publish-manual": "npm publish"
   },
   "peerDependencies": {
-    "prettier": "2.x",
-    "@trivago/prettier-plugin-sort-imports": "3.x"
+    "@trivago/prettier-plugin-sort-imports": "4.2.1",
+    "prettier": "3.0.3"
```

can we make these carets?
```suggestion
    "@trivago/prettier-plugin-sort-imports": "^4.2.1",
    "prettier": "^3.0.3"
```
Since we only really intend for this to be used by us, we don't _have_ to follow conventions for peerdeps, but it's generally not a good idea to pin a single version in `peerDependencies`, or even in `dependencies`